--- a/doc/mpdconf.example
+++ b/doc/mpdconf.example
@@ -82,7 +82,7 @@
 #bind_to_address		"any"
 #
 # And for Unix Socket
-#bind_to_address		"~/.mpd/socket"
+#bind_to_address		"/run/mpd/socket"
 #
 # This setting is the TCP port that is desired for the daemon to get assigned
 # to.
@@ -273,13 +273,13 @@
 #
 # An example of a pulseaudio output (streaming to a remote pulseaudio server)
 #
-#audio_output {
-#	type		"pulse"
-#	name		"My Pulse Output"
+audio_output {
+	type		"pulse"
+	name		"My Pulse Output"
 ##	server		"remote_server"		# optional
 ##	sink		"remote_server_sink"	# optional
 ##	media_role	"media_role"		#optional
-#}
+}
 #
 # An example of a winmm output (Windows multimedia API).
 #

