From b44204effc67e2d9dc3ae5e9b54beeec6961d92d Mon Sep 17 00:00:00 2001
From: Jamie Nguyen <j@jamielinux.com>
Date: Fri, 7 Nov 2014 12:21:24 +0000
Subject: [PATCH] Disable systemd controlgroup directives

---
 systemd/mpd.service.in | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/systemd/mpd.service.in b/systemd/mpd.service.in
index a72eb92..de91eb4 100644
--- a/systemd/mpd.service.in
+++ b/systemd/mpd.service.in
@@ -9,14 +9,5 @@ ExecStart=@prefix@/bin/mpd --no-daemon
 LimitRTPRIO=50
 LimitRTTIME=-1
 
-# move MPD to a top-level cgroup, as real-time budget assignment fails
-# in cgroup /system/mpd.service, because /system has a zero real-time
-# budget; see
-# http://www.freedesktop.org/wiki/Software/systemd/MyServiceCantGetRealtime/
-ControlGroup=cpu:/mpd
-
-# assign a real-time budget
-ControlGroupAttribute=cpu.rt_runtime_us 500000
-
 [Install]
 WantedBy=multi-user.target
-- 
2.1.0

