--- a/systemd/system/mpd.service.in
+++ b/systemd/system/mpd.service.in
@@ -23,7 +23,6 @@ LimitMEMLOCK=64M
 ProtectSystem=yes
 
 # more paranoid security settings
-NoNewPrivileges=yes
 ProtectKernelTunables=yes
 ProtectControlGroups=yes
 ProtectKernelModules=yes
--- a/systemd/user/mpd.service.in
+++ b/systemd/user/mpd.service.in
@@ -25,7 +25,6 @@ PrivateUsers=yes
 ProtectSystem=yes
 
 # more paranoid security settings
-NoNewPrivileges=yes
 ProtectKernelTunables=yes
 ProtectControlGroups=yes
 RestrictAddressFamilies=@address_families@
