--- a/src/decoder/plugins/WildmidiDecoderPlugin.cxx
+++ b/src/decoder/plugins/WildmidiDecoderPlugin.cxx
@@ -30,7 +30,7 @@ wildmidi_init(const ConfigBlock &block)
 {
 	const AllocatedPath path =
 		block.GetPath("config_file",
-			      "/etc/timidity/timidity.cfg");
+			      "/etc/timidity.cfg");
 
 	if (!FileExists(path))
 		throw PluginUnavailable{FmtBuffer<1024>("configuration file does not exist: {}",
